demos/interactive_face_detection_demo/cpp/README.md

```diff
@@ -69,42 +69,12 @@ omz_converter --list models.lst
 
 ## Running
 
-Running the application with the `-h` option yields the following usage message:
-
-```
-interactive_face_detection_demo [OPTION]
-Options:
-
-    -h                         Print a usage message
-    -i                         Required. An input to process. The input must be a single image, a folder of images, video file or camera id.
-    -loop                      Optional. Enable reading the input in a loop.
-    -o "<path>"                Optional. Name of the output file(s) to save.
-    -limit "<num>"             Optional. Number of frames to store in output. If 0 is set, all frames are stored.
-    -m "<path>"                Required. Path to an .xml file with a trained Face Detection model.
-    -m_ag "<path>"             Optional. Path to an .xml file with a trained Age/Gender Recognition model.
-    -m_hp "<path>"             Optional. Path to an .xml file with a trained Head Pose Estimation model.
-    -m_em "<path>"             Optional. Path to an .xml file with a trained Emotions Recognition model.
-    -m_lm "<path>"             Optional. Path to an .xml file with a trained Facial Landmarks Estimation model.
-    -m_am "<path>"             Optional. Path to an .xml file with a trained Antispoofing Classification model.
-    -d <device>                Specify a target device to infer on (the list of available devices is shown below). Default value is CPU. Use "-d HETERO:<comma-separated_devices_list>" format to specify HETERO plugin. Use "-d MULTI:<comma-separated_devices_list>" format to specify MULTI plugin. The application looks for a suitable plugin for the specified device.
-    -no_show                   Optional. Don't show output.
-    -r                         Optional. Output inference results as raw values
-    -t                         Optional. Probability threshold for detections
-    -bb_enlarge_coef           Optional. Coefficient to enlarge/reduce the size of the bounding box around the detected face
-    -dx_coef                   Optional. Coefficient to shift the bounding box around the detected face along the Ox axis
-    -dy_coef                   Optional. Coefficient to shift the bounding box around the detected face along the Oy axis
-    -fps                       Optional. Maximum FPS for playing video
-    -no_smooth                 Optional. Do not smooth person attributes
-    -no_show_emotion_bar       Optional. Do not show emotion bar
-    -u                         Optional. List of monitors to show initially.
-```
-
-Running the application with an empty list of options yields the usage message given above and an error message.
+Running the application with an empty list of options yields the usage message and an error message.
```

Comment:
Other authors put it like https://github.com/openvinotoolkit/open_model_zoo/pull/3104/files#diff-f988d5acc4d4e573b5410eb51c367a60cafdd2bda3f051c24c1d5c35c9defbbfR122
Let's align with them. Please do this also for noise_supression.

Reply:
Done